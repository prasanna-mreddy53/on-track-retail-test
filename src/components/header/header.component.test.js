import React from "react";
import Header from "./header.component";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(<Header />);

describe("Header component tests", () => {
  it("should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
