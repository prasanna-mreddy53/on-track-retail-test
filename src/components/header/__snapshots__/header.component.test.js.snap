// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component tests should match snapshot 1`] = `
<header
  className="App-header"
>
  <Navbar
    bg="primary"
    collapseOnSelect={false}
    expand={true}
    variant="dark"
  >
    <NavbarBrand
      href="#home"
    >
      OnTrack Test
    </NavbarBrand>
    <Form
      className="ml-auto"
      inline={true}
    >
      <FormControl
        className="mr-sm-2"
        onChange={[Function]}
        placeholder="Search"
        type="text"
      />
      <Button
        active={false}
        disabled={false}
        onClick={[Function]}
        variant="outline-light"
      >
        Search
      </Button>
    </Form>
  </Navbar>
</header>
`;
